.github/workflows/release.yml

```diff
@@ -9,7 +9,7 @@ on:
 env:
   BASE_IMAGE_NAME: plone/frontend
   PLATFORMS: linux/amd64,linux/arm64
-  IS_LATEST: false
+  IS_LATEST: true
```

Comment:
We need to also change this to false on the 17.x branch

Reply:
Just did that